charts/presidio/Chart.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: v1
 description: A context aware, born to the cloud, customizable data loss prevention service
 name: presidio
-version: unstable
-appVersion: unstable
\ No newline at end of file
+version: 1.0
```

Comment:
what is this version number? the latest is 0.1.3.

Reply:
this version is the chart's version. not presidio's

Reply:
No you can't. The chart depends on it
 https://github.com/Microsoft/presidio/blob/development/charts/presidio/templates/analyzer-deployment.yaml#L23